pkg/apis/tenancy/v1beta1/types.go

```diff
@@ -78,11 +78,10 @@ type WorkspaceStatus struct {
 	// can be found. This URL can be used to access the workspace with standard Kubernetes
 	// client libraries and command line tools.
 	//
-	// +required
 	// +kubebuilder:format:uri
-	URL string `json:"URL"`
+	URL string `json:"URL,omitempty"`
 
-	// Phase of the workspace (Initializing / Active / Terminating). This field is ALPHA.
+	// Phase of the workspace (Scheduling, Initializing, Ready).
```

Comment:
Do we not expect to have terminating/deleting workspaces?

Reply:
We never had. There is DeletionTimestamp for that.

Reply:
Note that this just documents what we have implemented for half a year.